README.md

```diff
@@ -16,23 +16,17 @@ npm install --save @chanzuckerberg/eds
 yarn add @chanzuckerberg/eds
 ```
 
-Import the EDS stylesheet and tokens somewhere in your app, e.g. an `init.ts` or `app.ts` file:
+## App Setup
+
+Import the EDS stylesheet and tokens somewhere in your app root, e.g. an `init.ts` or `app.ts` file:
 
 ```js
 import '@chanzuckerberg/eds/index.css';
 // optionally import EDS font faces
 // import '@chanzuckerberg/eds/fonts.css';
 ```
 
-EDS components are designed for the Graphik font, but you may use other fonts by re-defining the `--eds-font-family-primary` CSS property. We also surface an `--eds-font-size-base` property to set your base `rem` font size, eg:
-
-```css
-html {
-  font-size: var(--eds-font-size-base);
-}
-```
-
-### Tailwind Config
+### Tailwind Setup
```

Comment:
Removing this from the readme, since it's about theming. We will expand on this in the more detailed theming docs elsewhere so it's not just hanging out by itself